https/tls_config.go

```diff
@@ -164,8 +166,23 @@ func ConfigToTLSConfig(c *TLSStruct) (*tls.Config, error) {
 	return cfg, nil
 }
 
-// Listen starts the server on the given address. If tlsConfigPath isn't empty the server connection will be started using TLS.
+// AddFlags adds the flags used by this package to the Kingpin application.
+// To use the default Kingpin application, call AddFlags(kingpin.CommandLine)
+func AddFlags(a *kingpin.Application) {
```

Comment:
Let's call this `AddKingpinFlags`, as we could include other flag library support in the future.